[READY FOR REVIEW] PPIE version of Inclusive Events chapter

BrainonSilicon opened the pull request:
<!--
Please complete the following sections when you submit your pull request. You are encouraged to keep this top level comment box updated as you develop and respond to reviews. Note that text within html comment tags will not be rendered.
-->
### Summary

<!-- Describe the problem you're trying to fix in this pull request. Please reference any related issue and use fixes/close to automatically close them, if pertinent. For example: "Fixes #58", or "Addresses (but does not close) #238". -->

This is the first draft of the PPIE version of the Inclusive Events chapter. 

The chapter contains guidance and recommendations for how to design an event to be inclusive and engaging for patients and members of the public who are part of a research project's  "PPIE" work packages. 

PPIE stands for Public and Patient Involvement and Engagement. You can read more about PPIE [here](http://hpruezi.nihr.ac.uk/public-involvement/). 

The chapter is a collection of best practices and recommendations developed by PPIE professionals and contributors to help future projects run inclusive and ethical PPIE events.  

It is written in tandem with an Accessibility focused version of the chapter which focuses on accessibility requirements and disability inclusion. For more information about the two chapters, please refer to Issue #3307. 


### List of changes proposed in this PR (pull-request)

<!-- We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

* Added content to chapter about running and hosting PPIE events
* Initializing update with `main` branch content
* File name changes from `inclusive-events` to `ppie-events`
* Changed content from the initial chapter draft to remove accessibility-specific information 
* Rephrased chapter to focus on PPIE-specific recommendations
* Added references to book/website/_bibliography/references.bib that were cited in the chapter content
* Added "Plain Language" and definition to the glossary (book/website/afterword/glossary.md)
* Added files to the _toc (table of contents)


### What should a reviewer concentrate their feedback on?

<!-- This section is particularly useful if you have a pull request that is still in development. You can guide the reviews to focus on the parts that are ready for their comments. We suggest using bullets (indicated by * or -) and filled checkboxes [x] here -->

* **Reviewers should focus their review on the following files:**
  * `book/website/collaboration/ppie-events.md`
  * `book/website/collaboration/ppie-events/ppie-events-planning.md`
  * `book/website/collaboration/ppie-events/ppie-events-location.md`
  * `book/website/collaboration/ppie-events/ppie-events-schedules.md`
  * `book/website/collaboration/ppie-events/ppie-events-comms.md`
  * `book/website/collaboration/ppie-events/ppie-events-socials.md`
  * `book/website/collaboration/ppie-events/ppie-events-tech.md`
  * `book/website/_bibliography/references.bib`
  * `book/website/afterword/glossary.md`
  * `_toc.yml`

When reviewing these files please review for 

- [ ] content
- [ ] is anything missing?
- [ ] spelling mistakes and grammar

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.

### Other Information

When reviewing please keep in mind that accessibility requirements and discussion regarding disability is not featured in this version of the chapter. 

Many sections from the original draft have been removed, and you can review a comparison of the headings and sections between drafts in the [heading_comparison_inclusive_events_chapters.csv](https://github.com/the-turing-way/the-turing-way/files/14384517/heading_comparison_inclusive_events_chapters.csv) file.
- The Accessibility focused version of the chapter will go into these topics in more detail.

KirstieJane commented:
What a lot of work @BrainonSilicon !! Well done!! 

I've added myself as a reviewer too ❤️

aleesteele commented on .all-contributorsrc:
```diff
@@ -254,7 +254,8 @@
         "ideas",
         "doc",
-        "review"
+        "review",
```
Oh no @BrainonSilicon - I think rebasing the branch accidentally committed your changes!

gedankenstuecke replied:
I think the problem is that `WIP-Inclusive-Events-chapter-Content` (the branch that this PR aims to merge from) was rebased at some point to pull more commits from `main`, while the branch that's the target of this PR (`WIP]Inclusive-Events-chapter`) is further behind `main` in comparison. 

I'm not sure what the best practice would in such a case. Would it be to rebase both PRs to be on the same level of the `main` branch?

BrainonSilicon replied:
Massive thanks to @gedankenstuecke and @da5nsy for figuring it out (and @JimMadge for the Friday knowledge drop on Slack to guide the way)

Some learnings 
- PRs work to specific commits and rebases seems to undo this in some way (so be careful when rebasing) 
- best to add references in random places

da5nsy commented on book/website/_bibliography/references.bib:
```diff
@@ -2109,6 +2116,31 @@ @article{Bonvoisin2018
+	author = {National Institude for Health and Care Research},
+	title = {Different experiences: A framework for considering who might be involved in research},
+	url = {https://www.nihr.ac.uk/documents/different-experiences-a-framework-for-considering-who-might-be-involved-in-research/27387}
+  	urldate = {2023-11-10},
```
```suggestion
  	urldate = {2023-11-10}
```

da5nsy commented on book/website/_bibliography/references.bib:
```diff
@@ -2109,6 +2116,31 @@ @article{Bonvoisin2018
+	author = {Camden Coalition},
+	title = {Understanding preferred language for people with lived experience},
+	url = {https://camdenhealth.org/blog/understanding-preferred-language-for-people-with-lived-experience/}
+  	urldate = {2023-11-10},
```
```suggestion
  	urldate = {2023-11-10}
```

da5nsy commented on book/website/_bibliography/references.bib:
```diff
@@ -2109,6 +2116,31 @@ @article{Bonvoisin2018
+	year = {2021},
+	author = {National Institude for Health and Care Research},
+	title = {Different experiences: A framework for considering who might be involved in research},
+	url = {https://www.nihr.ac.uk/documents/different-experiences-a-framework-for-considering-who-might-be-involved-in-research/27387}
```
```suggestion
	url = {https://www.nihr.ac.uk/documents/different-experiences-a-framework-for-considering-who-might-be-involved-in-research/27387},
```

da5nsy commented on book/website/_bibliography/references.bib:
```diff
@@ -2109,6 +2116,31 @@ @article{Bonvoisin2018
+	month = {Jun},
+	author = {Camden Coalition},
+	title = {Understanding preferred language for people with lived experience},
+	url = {https://camdenhealth.org/blog/understanding-preferred-language-for-people-with-lived-experience/}
```
```suggestion
	url = {https://camdenhealth.org/blog/understanding-preferred-language-for-people-with-lived-experience/},
```

da5nsy commented:
There's a few warnings coming up during build to do with re-used labels and pages that aren't included in the toc:

```
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:2: WARNING: duplicate label cl-inclusive-events-checklist, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:9: WARNING: duplicate label cl-inclusive-events-checklist-focus-session, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:16: WARNING: duplicate label cl-inclusive-events-checklist-interest-group, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:23: WARNING: duplicate label cl-inclusive-events-checklist-seminar, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:30: WARNING: duplicate label cl-inclusive-events-checklist-workshop, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:37: WARNING: duplicate label cl-inclusive-events-checklist-conference, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-planning.md:150: WARNING: Duplicate explicit target name: "cl-ppie-events-timeline-planning".

/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-resources.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-comms.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-location.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-planning.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-resources.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-schedules.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-socials.md: WARNING: document isn't included in any toctree
/home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-tech.md: WARNING: document isn't included in any toctree
```

BrainonSilicon reviewed (commented):
Hi @LizHareDogs 

Thank you so much for these additions and suggestions. I reviewed the content and have made a few comments for discussion. 

I think the next steps are to get the draft out to the wider PPIE community as well as the TTW community for review!

BrainonSilicon commented:
Hi @EstherPlomp the content here is ready to be reviewed if and when you have time :)

BrainonSilicon commented:
> There's a few warnings coming up during build to do with re-used labels and pages that aren't included in the toc:
> 
> ```
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:2: WARNING: duplicate label cl-inclusive-events-checklist, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:9: WARNING: duplicate label cl-inclusive-events-checklist-focus-session, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:16: WARNING: duplicate label cl-inclusive-events-checklist-interest-group, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:23: WARNING: duplicate label cl-inclusive-events-checklist-seminar, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:30: WARNING: duplicate label cl-inclusive-events-checklist-workshop, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md:37: WARNING: duplicate label cl-inclusive-events-checklist-conference, other instance in /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-planning.md:150: WARNING: Duplicate explicit target name: "cl-ppie-events-timeline-planning".
> 
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-resources.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-comms.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-location.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-planning.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-resources.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-schedules.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-socials.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-tech.md: WARNING: document isn't included in any toctree
> ```

I think I've fixed these now :)

BrainonSilicon commented:
I've now added the pages to the toc 

The only page left is the checklist which does not current have content. Waiting for a content review before adding. 
- version 1 of the chapter cane be launched/merged without the checklist file.

> There's a few warnings coming up during build to do with pages that aren't included in the toc:
> 
> ```
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-checklist.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/inclusive-events/inclusive-events-resources.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-checklist.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-comms.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-location.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-planning.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-resources.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-schedules.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-socials.md: WARNING: document isn't included in any toctree
> /home/runner/work/the-turing-way/the-turing-way/book/website/collaboration/ppie-events/ppie-events-tech.md: WARNING: document isn't included in any toctree
> ```

aleesteele commented on book/website/collaboration/ppie-events.md:
```diff
@@ -0,0 +1,60 @@
+PPIE is done both to ensure that research is aligned with the interests of people affected by it, and to improve the quality of research by including lived knowledge and perspectives. 
+As part of this involvement and engagement, researchers and research staff will need to plan events that are inclusive and accessible for people with lived experience, and delivered in a way that avoids presenting barriers to people with lived experience taking part equitably.
+
+"Lived experience" can be a range of things such as a disability, long-term health condition, bereavement, having worked in a specific profession, or the lived experience of caring responsibilities. 
```
```suggestion
"Lived experience" as it is used within the research context can be a range of things such as a disability, long-term health condition, bereavement, having worked in a specific profession, or the lived experience of caring responsibilities. 
```
Adding a gentle suggestion here for some qualifying language. Should this be added to the glossary?

BrainonSilicon replied:
Good suggestion! I think it would be a good idea to enter even a basic definition in. 

Do you think this sentence should also reference how the phrase lived experience is used/means within a policy, social, and health care settings too? Or do we need to narrow it down to just be relevant to the research experience?

aleesteele commented on book/website/collaboration/ppie-events.md:
```diff
@@ -0,0 +1,60 @@
+
+Thus, your considerations need to cover both the potential disability-related barriers experienced by your attendees as well as the barriers that would be experienced by your attendees with lived experiences such as parental responsibilities, cultural differences, or minority status. 
+
+This chapter focuses on the latter aspects, specific to involving people with lived experiences, for the more general and equally important considerations around accessibility – that affect participants regardless of whether they are members of the public or not, please refer to specific accessibility resources.
```
```suggestion
This chapter focuses on the latter aspects, specific to involving people with lived experiences, for the more general and equally important considerations around accessibility – that affect participants regardless of whether they are members of the public or not, please refer to specific accessibility or access-centered resources.
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-checklist.md:
```diff
@@ -0,0 +1,60 @@
+(cl-ppie-events-checklist)=
+# Checklist
```
Is this something you will list out later - with the action points?

BrainonSilicon replied:
Yes - it was hard to draft the checklist when the content kept changing so it is going to put on hold until the content has been finalized and signed off on :)

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+- Why are you trying to reach them?
+- What information do you need to include?
+- When do your attendees with lived experience need the information?
+- How will you convey the information?
```
```suggestion
- How will you convey the information to them?
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+For this chapter, "Communications" refers to the information you share with potential attendees, the timings and schedule on which you communicate it, the channels you will use, and the content and information about the event. 
+
+When developing a communications strategy, in addition to identifying the goals for your event, you will need to consider:
+- Who are you trying to reach?
```
```suggestion
- Who are you trying to reach more broadly?
```
Just a suggestion - is this a broader question about reach, asking about attendees with lived experiences as a subset?

BrainonSilicon replied:
It's more about the audience in general - the suggestions are more guidance for how a communications strategy can be created - i.e. who is the target audience for a specific piece of information.

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+Your "Save the Date" should include an overview of honorarium or reimbursement support, the date, and the approximate location of the event such as Zoom, or the name of the city where the event will take place. 
+It should also reference how long the event is likely to be, for example, if it's a multi-day event or a few hours. 
+
+This can be helpful for attendees with caring responsibilities, who may need to hire and arrange alternative care support in order to attend your event, or could be placed at a significant financial disadvantage if they travel and attend. 
```
Repeats phrasing used earlier on (line 37)

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+Depending on your event and where you host online information, you should ensure that the digital home for your event covers this information:
+- How to attend (joining links for online content and location information for in-person events).
+- When and where your event is.
+- What accessibility practices will be implemented for your event
```
```suggestion
- What accessibility practices will be implemented for your event.
```
Adding the period as the other points have it!

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+
+You can also prepare the speakers by asking how they would like to interact with any questions that are asked. 
+Many research talks often follow a "lecture" format where a speaker talks on a topic and then takes 2-3 questions from the audience. 
+Consider asking your speaker to break their talk into sections to allow for questions at different points in the session, or if your event is a lecture, consider using software to gather questions as the session progresses. 
```
```suggestion
Consider asking your speaker to break their talk into sections to allow for questions at different points in the session, or if your event is a lecture, consider using software to gather questions as the session progresses (i.e. Slido, Mentimeter). 
```
Just adding some suggested questions here

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+
+This can help in two core ways, 1) supporting speaking in developing both psychologically and physically accessible content, and 2) ensuring that your event's content is accessible to a diverse audience.
+
+Firstly, consider how you will support speakers in developing inclusive material that is pitched to the correct audience, which in this case will be members of the public and people with lived experience). 
```
```suggestion
Firstly, consider how you will support speakers in developing inclusive material that is pitched to the correct audience, which in this case will be members of the public and people with lived experience. 
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+However, being able to engage in an event’s content and simultaneously read notes often requires attendees to have access to a secondary screen or device that can be used for typing, reading, and to view both written and presented material. 
+Many members of the public may not have multiple screens set up in their homes, and they may be attending online events on their phones. 
+
+Ensure that if you offer live notetaking at your event that your attendees are able to access both the session content and the notes, and that technology you use is one that your attendees are comfortable using and getting access to. 
```
```suggestion
Ensure that if you offer live notetaking at your event that your attendees are able to access both the session content and the notes, and that technology you use is one that your attendees are comfortable using and getting access to. Some examples include:
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+Reflect on how, and where, your research field uses jargon that might not be understandable to the general public. 
+Although you may be able to assume that many people with lived experience are familiar with any common acronyms or jargon used, it can benefit the inclusion of all attendees and experience levels if the amount of jargon used can be minimized. 
+For example, researchers, clinicians and people with a lived experience of arthritis may be more familiar with terms and medication names related to arthritis and associated conditions. 
+However, many attendees may be unfamiliar with these terms can can feel excluded from conversation and event content that uses them 
```
```suggestion
However, many attendees may be unfamiliar with these terms can can feel excluded from conversation and event content that uses them.
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+    * The Event organiser. This may be the designated point of contact in an emergency.
+    * A person who is able to provide Accessibility information and help if you are able to support it for your event.
+      Try to make this contact available as soon as you announce the event, even if it is setting up a designated inbox.
+    * Any Catering or dietary requirements.
```
```suggestion
    * Any catering or dietary requirements.
```

BrainonSilicon commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+If your event has online or streamed sessions, consider developing an information pack and working with your speakers and event facilitators to ensure that content is accessible and inclusive of all attendees. 
+You can use this pack to highlight information about the audience that the speaker should know, and provide guidance on how to effectively communicate to an audience of people with lived experience.
+In this resource, you can model how to talk with respect about the people with the type of lived experience that is the topic of your event, using the vocabulary the community chooses and avoiding assumptions related to that topic.
+Many early career researchers may not have received formal training in scientific communication and working with the public, so it can be helpful to emphasize the importance of using plain language and speaking clearly. 
```
![5cd381hhhrdim5ja](https://github.com/the-turing-way/the-turing-way/assets/42813259/55eb1f6b-63a4-4dea-ab7e-2336cbfca1f8)

BrainonSilicon reviewed (commented):
Thanks for the review Anne!

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-comms.md:
```diff
@@ -0,0 +1,363 @@
+    * A person who is able to provide Accessibility information and help if you are able to support it for your event.
+      Try to make this contact available as soon as you announce the event, even if it is setting up a designated inbox.
+    * Any Catering or dietary requirements.
+      Because dietary requirements are protected information, not all members of an organizing team will have access to it or be able to answer questions. 
```
It might be worth adding in a previous section some information about "protected characteristics" as they are used in the UK & USA for example, but not not apply more broadly, or being known as a term.

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-location.md:
```diff
@@ -0,0 +1,74 @@
+There is more guidance on how to choose an Accessible location in the Accessible Events Guide. 
+This can be helpful to both check the maintenance and layout of the space, and can also help you plan your event set-up such as where to put signs directing attendees to your event.
+
+Through a Universal Design lens, a venue that is able to support a diverse attendee list will be best for any event that involves both a research and general public audience. 
```
```suggestion
Through a [Universal Design](https://universaldesign.ie/about-universal-design) lens, a venue that is able to support a diverse attendee list will be best for any event that involves both a research and general public audience. 
```

BrainonSilicon replied:
should we link to an external website - or the section which discusses universal design in the chapter?

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-location.md:
```diff
@@ -0,0 +1,74 @@
+A large hall with high ceilings may cause echoing and may not be suitable for discussion-based sessions. Similarly, a small room with limited ventilation may not be suited for long sessions with a large number of attendees. 
+These considerations will benefit all events, however may be more important to consider for events that are intended for people with lived experience. 
+It is important to ensure that a space can support the involvement of all its attendees. 
+This can be considering how to layout tables (circular is best) so that all those seated are able to listen and contribute equally to the discussion, and can also be considering how the space itself will be laid out. 
```
```suggestion
This can be considering how to lay out tables (circular is best) so that all those seated are able to listen and contribute equally to the discussion, and can also be considering how the space itself will be laid out. 
```
Maybe also ensuring sufficient seating areas, as well as assigned quiet and/or sensory spaces?

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-location.md:
```diff
@@ -0,0 +1,74 @@
+
+
+(cl-ppie-events-location-during-event)=
+## During the event
```
Are sensory maps something that you think should be added to this section? I've seen them commonly used in making more spaces friendly for autistic participants? What do you think?

Maybe: Consider creating a sensory map, which is designed to help attendees understand and plan for the sensory experiences they may have throughout the space.

For example: https://www.sciencemuseum.org.uk/sites/default/files/2022-07/Sensory-Map-Science-Museum-July-2022.pdf

BrainonSilicon replied:
I had left those suggestions to the accessibility chapter :)

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-planning.md:
```diff
@@ -0,0 +1,329 @@
+
+At each stage, your planning documents should reflect an emphasis on accessible and inclusive universal design. 
+
+(cl-pppie-events-initial-planning)=
```
```suggestion
(cl-ppie-events-initial-planning)=
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-planning.md:
````diff
@@ -0,0 +1,329 @@
+
+
+```{note}
+If you are able to collaborate with  members of the community, please consider how you will recognize or reimburse the time and contributions made.
````
```suggestion
If you are able to collaborate with members of the community, please consider how you will recognize or reimburse the time and contributions made.
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-resources.md:
```diff
@@ -0,0 +1,41 @@
+## What to Learn Next
+
+> **Recommend other chapters in _The Turing Way_ book that readers should read next as a "level up", and explain why.**
+* {ref}`cm-lay-summaries` contains information and guidance on writing plain langugage summaries that canhelp guide the development of your plain langugage resources
```
```suggestion
* {ref}`cm-lay-summaries` contains information and guidance on writing plain language summaries that can help guide the development of your plain language resources
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-resources.md:
```diff
@@ -0,0 +1,41 @@
+
+> **Recommend other chapters in _The Turing Way_ book that readers should read next as a "level up", and explain why.**
+* {ref}`cm-lay-summaries` contains information and guidance on writing plain langugage summaries that canhelp guide the development of your plain langugage resources
+* {ref}`cl-organising-conferences` contains a checklist and general guidance for organizing a conferences which supplements this chapter if you are planning a longer or multi-day event
```
```suggestion
* {ref}`cl-organising-conferences` contains a checklist and general guidance for organizing a conference which supplements this chapter if you are planning a longer or multi-day event
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-resources.md:
```diff
@@ -0,0 +1,41 @@
+> **Recommend other chapters in _The Turing Way_ book that readers should read next as a "level up", and explain why.**
+* {ref}`cm-lay-summaries` contains information and guidance on writing plain langugage summaries that canhelp guide the development of your plain langugage resources
+* {ref}`cl-organising-conferences` contains a checklist and general guidance for organizing a conferences which supplements this chapter if you are planning a longer or multi-day event
+* {ref}`cl-event-tools` contains suggestions and reccomendations for tools that can help with task management, collaborative note taking, live polls, and surveys which can be used to support your event
```
```suggestion
* {ref}`cl-event-tools` contains suggestions and recommendations for tools that can help with task management, collaborative note taking, live polls, and surveys which can be used to support your event
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-resources.md:
```diff
@@ -0,0 +1,41 @@
+* {ref}`cm-lay-summaries` contains information and guidance on writing plain langugage summaries that canhelp guide the development of your plain langugage resources
+* {ref}`cl-organising-conferences` contains a checklist and general guidance for organizing a conferences which supplements this chapter if you are planning a longer or multi-day event
+* {ref}`cl-event-tools` contains suggestions and reccomendations for tools that can help with task management, collaborative note taking, live polls, and surveys which can be used to support your event
+* {ref}`cl-remote-collab-guidelines` provides guidance on running remote meetings and has reccomendations that can help shape an online or remote event held for people with lived experience
```
```suggestion
* {ref}`cl-remote-collab-guidelines` provides guidance on running remote meetings and has recommendations that can help shape an online or remote event held for people with lived experience
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-location.md:
```diff
@@ -0,0 +1,74 @@
+When assisting a disabled person, ask them how you can do this because it varies between individuals.
+Do not touch, grab, or move people with disabilities, their mobility aids, or service dogs without permission.
+
+It is also important to be cognizant that attendees who are more familiar with the layout of University campuses may feel more comfortable navigating university-based locations. 
```
```suggestion
It is also important to be cognizant that attendees who are more familiar with the layout of university campuses may feel more comfortable navigating university-based locations. 
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-location.md:
```diff
@@ -0,0 +1,74 @@
+This may mean putting up posters or banners and decorating areas of the space to allow attendees to feel a sense of belonging and identifying where your event spaces are located within a larger venue.
+
+If you are able to provide attendees with a map of the venue, make sure to identify how to navigate the location, where parking and restrooms are located, where to find staff members, and where all important locations will be (such as kitchen, break out rooms, and registration). 
+Also ensure that the location of any emergency supplies such as an AED, epi-pen, first aid kit and a member of staff who will be able to act in the event of an emergency is marked on the map. 
```
```suggestion
Also ensure that the location of any emergency supplies such as an automated external defibrillator (AED), epi-pen, first aid kit and a member of staff who will be able to act in the event of an emergency is marked on the map. 
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-schedules.md:
```diff
@@ -0,0 +1,318 @@
+### Session Duration
+Your schedule should take into account breaks and meal times, and carefully plan the length of its sessions. 
+
+If your event takes place in person, make sure to allow for sufficient time for people with lived experience and disabilities affecting mobility to move between locations. 
```
```suggestion
If your event takes place in person, make sure to allow for sufficient time for people with lived experiences and/or disabilities affecting mobility to move between locations. 
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-schedules.md:
```diff
@@ -0,0 +1,318 @@
+
+The guidance you provide will be different depending on your event. 
+For example, a public lecture may ask attendees to submit questions for the speaker at the end of the lecture. 
+For this example, indicate in your guidance how long there will be for questions, how to use any online software you may be using to collect audience questions, and also how many questions you are likely going to answer. 
```
```suggestion
For this example, indicate in your guidance how much time will be used for questions, how to use any online software you may be using to collect audience questions, and also how many questions you are likely going to answer. 
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-schedules.md:
```diff
@@ -0,0 +1,318 @@
+For this example, indicate in your guidance how long there will be for questions, how to use any online software you may be using to collect audience questions, and also how many questions you are likely going to answer. 
+This can help avoid any feeling attendees may have about being ignored or not heard if their question is not addressed. 
+Ensure that the moderator is taking questions from a diverse subset of the audience.
+
```
```suggestion
```
Empty spaces here

aleesteele commented:
Reviewed everything besides schedules, socials, and tech today. During the Accessibility working group, we also mapped out the branches/issues needed to merge across different resources (including the #3426). Will finish up my review tomorrow, @BrainonSilicon! Thank you for your brilliant work on this chapter and all your hard work and collaboration with @LizHareDogs 🙏

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-socials.md:
```diff
@@ -0,0 +1,121 @@
+Depending on the size and timings of your event, you can consider setting prompts such as "find a fellow attendee who has the same favourite movie as you", or "introduce yourself to 5 people you have never met before, and find 1 thing you have in common". 
+Prompts such as these can provide attendees with a goal for the session and help attendees to start conversations with one another. 
+
+Good conversation prompts or ice breaker questions are both interesting for someone to think about and answer themselves, and also make attendees curious to hear everyone else’s answer.
```
I've also seen "stickers" or "name tags" that people can wear that prompt conversation... Like, "ask me about..."!

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-socials.md:
```diff
@@ -0,0 +1,121 @@
+## Conclusion
+Social events can often be uncomfortable and daunting for many people for a variety of reasons.
+
+By providing a guide and establishing a structured approach to any social activities you will be able to reduce any potential anxieties that attendees may have and support all attendees being comfortable in fully engaging with your event.
```
```suggestion
By providing a guide and establishing a structured approach to any social activities you may be able to reduce potential anxieties that attendees may have and support all attendees being comfortable in fully engaging with your event.
```
Suggestion to 'may' vs 'will'

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-tech.md:
```diff
@@ -0,0 +1,100 @@
+This section focuses on considerations and recommendations around using different technologies to support the inclusion of people with lived experience at in-person, hybrid, or online events. 
+
+(cl-ppie-events-hybrid)=
+## Hybrid as an Accessibility and Inclusion Strategy
```
```suggestion
## Hybrid Events as an Accessibility and Inclusion Strategy
```

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-tech.md:
```diff
@@ -0,0 +1,100 @@
+### Reaching attendees without email access
+Your event may engage communities and people with lived experience who have barriers to accessing email or the internet regularly. 
+
+Consider how you will keep in contact with these individuals, and what additional resources and safeguarding your organizers and staff members may need. 
```
Do these events usually have safeguarding protocols? I've seen some official plans, but not sure how it usually is in a PPIE context. Ignore me if not!

aleesteele commented on book/website/collaboration/ppie-events/ppie-events-tech.md:
```diff
@@ -0,0 +1,100 @@
+Consider developing and providing tutorials and guides for each different technology tool that your attendees will interact with at your event. 
+This is important for both in-person and especially so for online or hybrid events. 
+
+This includes guides for technology such as [Zoom](https://zoom.us/), [Slido](https://www.slido.com), [Slack](https://slack.com/), and any interactive software such as Miro or Microsoft Whiteboards. 
```
Oh, I know I added a comment above about Slido links. They're here! Maybe having them linked in both places could work well?

BrainonSilicon commented:
Copying across @LizHareDogs's review comments from her Pull Request so that future reviewers can see the suggestions made: https://github.com/the-turing-way/the-turing-way/pull/3584


> ### Summary
> 
> This is an edit to hopefully
> 
>     * use language that is respectful and positive about people with lived experience
> 
>     * make things concise and avoid repetition
> 
>     * add tips based on my experience
> 
>     * remove the service dog tip, which can be replaced with tosmething that doesn't involve making things more difficult for service dog users
> 
> 
> Addresses PR [#3550](https://github.com/the-turing-way/the-turing-way/pull/3550)
> 
> Fixes #
> ### Description of Edits
> #### Hoping for more feedback and discussion from the disability community!*
> 
>     * I changed recommendations of "a reprsentative" from the community to the plural "representatives" for organizing for a few reasons
>       
>       * people are diverse; multiple perspectives are better
>       * concentrating the power in one person, even if it is a person with lived experience, can lead to negative outcomes
>       * being the only representative is _hard_ if you care about diversity and inclusion (ask me how I know!)
> 
>     * I changed "impairment" to disability. Some groups of disabled people really don't like "impairment"-- like deaf people who think it denies the positive aspects of having a culture and language. While some of them also don't like the term "disability" it seems to be better accepted than "impairment."
> 
>     * I tried to make the people with lived experience sound more like active contributors and people with agency, rather than people in constant need of support
>       
>       * In the context of helping speaker with lived experience prepare their talks, I took out the phrase about "setting them up for success" because it sounded patronizing.
>       * I removed wording about telling people what to avoid talking about. We can't invite people in and then censor them.
>       * Reasoning: this is a place to implicitly model respect for people with lived experience rather than a patroniizing, custodial, or infantalizing tone. **Remember that people with lived experience are welcome to read this document!**
>       * also in the Communications, this is uncomfortable to read:
>       
>       > If your event involves people with lived experience, you should consider how to set your speakers, presenters, and facilitators up for success.
>       > You will need to identify and invite your speakers, regardless of their lived experience, well in advance of the event and provide them with support so that they can plan and deliver a successful session.
> 
> 
> I'm trying to figure out how to edit this. It seems to come from an assumption that people with lived experience are inferior and likely to fail without patronizing intervention.
> 
>     * I have tried to get rid of the frequent use of "needs" which are attributed to the people with lived experience. The "need" is with the whole community, so rather than talking about needy people, I changed the focus to _practices_ we implement for inclusion.
> 
>     * I changed "the voice of people with lived experience" to "perspective of people with lived experience" because deaf people have pointed out that that metaphor is exclusive of people who don't express themselves with their voice.
> 
>     * The word "challenge" is problematic because it's used as an avoidant euphemism around disability issues, so I tried to re-word things when that came up
> 
>     * in the section on events planning, I added mentions of medical ableism and racism because I have seen this at these kinds of events. I added a sentence about how the goals of researchers can be different from people with lived experience-- especially around the idea of being "cured" which is offensive for people who think their identities are valid.
> 
>     * I changed the phrase "low vision or visual impairments" because it's kind of redundant. I changed it to "low vision or blindness" because, although these terms can overlap, the community often uses "blind" wo mean "without useful vision" and "low vision" to mean having some kind of useful vision. I
> 
>     * Communication section: I took out people with screen readers when talking about how some people prefer offline information, since use of screen readers is so closely tied to technology. This might have been relative when information was given out on disks, but now a print paper is relative inaccessible for blind people.
> 
>     * In the Communications section, I changed the order because I thought putting "experts by experience" last in the list showed just the kind of hierarchy we're trying to avoid: For example "researchers, clinicians, experts by experience, policymakers, and funding bodies."
> 
>     * I have removed the tip about sending an email about a participant's guide dogs. People with gservice animals experience discrimination, which was not acknowledged. While it is important to consider people with fears and allergies, allergy is also often used as an excuse to exclude service dog handlers. This sample email is an example of how to give people permission to discriminate, and I don't think we need to rely on something like this as an example.
> 
> 
> ### What should a reviewer concentrate their feedback on?
> 
> It would be great if people could help with
> 
>     * more perspectives from people with lived experience
>     * more tips for people with lived experience
>     * inclusive and respectful tone
>     * American spelling (sorry!)

KirstieJane commented:
@JimMadge @aleesteele - any idea why this PR doesn't build a preview? I can see that Jim has been trying to fix it 😬

BrainonSilicon commented:
> @JimMadge @aleesteele - any idea why this PR doesn't build a preview? I can see that Jim has been trying to fix it 😬 

@KirstieJane previews are only shown for PRs that go to main. There was a PR (3583) that was opened by accident and had overwrote this PR which was why we had some previews before.

BrainonSilicon commented:
Hiya @JimMadge (or @the-turing-way/infrastructure-working-group team) 

I wanted to check if we intended to keep some work we did while playing around fixes the branches a few weeks ago. 

Changes
---
In this PR the `.github/workflows/ci.yml` file is edited so that some of the checks run on PRs that aren't just ones that push to main. 
- Did we intend to keep this change (we don't need to!!) or should I revert it? 

Considerations
---
- Will the change run down the deployment minutes allocation that TTW has?
- Does it improve the workflow of folks contributing to TTW?

bsipocz commented:
> In this PR the .github/workflows/ci.yml file is edited so that some of the checks run on PRs that aren't just ones that push to main.
>
> Did we intend to keep this change (we don't need to!!) or should I revert it?

Yes, those changes should be reverted and this PR should contain no changes to the CI config.

I also wonder whether this PR is indeed ~300 commits strong or something else is going on with its commit history.

BrainonSilicon commented:
> > In this PR the .github/workflows/ci.yml file is edited so that some of the checks run on PRs that aren't just ones that push to main.
> > Did we intend to keep this change (we don't need to!!) or should I revert it?
> 
> Yes, those changes should be reverted and this PR should contain no changes to the CI config.
> 
> I also wonder whether this PR is indeed ~300 commits strong or something else is going on with its commit history.

Hi @bsipocz 

Thank you for confirming! I'll fix that now :) 

On the other note, the chapter has been a massively massively iterative process and it is now almost 300 commits deep. A lot of them have been related to small wording changes and phrasing adjustments.
The draft word count was almost 30,000 before starting review and it's more a case of not always batching commits and working with the reviews directly in the PR instead of using the branches.

BrainonSilicon reviewed (commented):
The chapter is starting to come together!

BrainonSilicon reviewed (commented):
made phase 1 of additions to the location section based on community feedback

BrainonSilicon commented:
In preparation for the chapter being merged during CW24. This PR is now being merged into the [`WIP-Inclusive-Events-chapter`](https://github.com/the-turing-way/the-turing-way/tree/WIP%5DInclusive-Events-chapter) branch 🚀 

Thank you everyone for your reviews and feedback so far. 

It's awesome to see how the chapter has progressed.
